src/runtime/substitution.cc

```diff
@@ -1880,6 +1884,15 @@ Graph *GraphSearchHelper::construct_graph() {
   return graph;
 }
 
+/**
+ * @brief Unity search algorithm main entrance.
+ *
+ * @param[in] budget Not used
+ * @param[in] only_data_parallel Not used
+ * @param[out] best_graph The best possible PCG after optimization
+ * @param[out] optimal_views The corresponding device placement views of the
+ * best graph
```

Comment:
```suggestion
 *             best graph
```

Reply:
Is there a doxygen autoformatter?

Reply:
Hmm, clang-format will format the doxygen comment as well. I think we should just rely on it. I'm not aware of a doxygen formatter.

Reply:
Huh, do we currently have doxygen autoformatting enabled in the clang format config? This formatting seems suboptimal

Reply:
I'm not quite sure if clang-format has an option to auto format doxygen comments? @gabrieleoliaro Do you know anything about doxygen comment formatter? Thx